src/Pages/Login.jsx

```diff
@@ -1,7 +1,38 @@
+import { Header , Footer} from "../Components/index.js";
+import { Link } from "react-router-dom";
+
 export default function Login() {
 
-    return (
-        <>
-            <h1>This is Login</h1>
-        </>)
+    return (<div className="flex-column stick-bottom">
+
+        <Header />
+
+        <div className="container flex-center-column">
+
+            <h2 className="text-xl login-header">Login</h2>
+            <div className="login-form flex-center-column box-shadow">
+
+                <div className="flex-column">
+                    <label for="email">Email:</label>
+                    <input className="login-input" type="text" id="email" required/>
+                </div>
+                <div className="flex-column">
+                    <label for="password">Password:</label>
+                    <input className="login-input" type="password" id="password" required/>
+                </div>
+                <div>
+                    <input type="checkbox" id="remember" />
+                    <label for="confirm">Remember me</label>
+
+                    <span className="signup">Don't have an account?, <Link classNameName="signup-link" to="/signup">Signup</Link></span>
```

Comment:
in link it should be className instead of classNameName. Fix typo

Reply:
Thanks for pointing out. Fixed it in the latest commit.